game/hud/src/components/PlayerStatusComponent/components/Pills/index.tsx

```diff
@@ -6,6 +6,14 @@
 
 import * as React from 'react';
 
+// Calculate percentage to two decimal places
+function P2DP(v: number, m: number) {
```

Comment:
I don't like abbreviated function names like this, please give functions a descriptive name in the future.